### PR TITLE
SSS-60 Depth 20 feature added

### PR DESCRIPTION
**Pull request by Amol-Bhagwat-187**
Changes done in constant.js and websocket2.0.js. In websocket2.0.js DEPTH() function is added to get depth 20 data.

### CODE REVIEW

File: example/test.js [hunk 1]
```diff
@@ -139,12 +139,12 @@ let smart_api = new SmartAPI({
    		// 		});
 
 		// search Scrip Methods
-		smart_api.searchScrip({
```
Comment (anurag-amx) [inline example/test.js:142]: why this code commented?

File: lib/websocket2.0.js [hunk 2]
```diff
@@ -116,6 +121,11 @@ let WebSocketV2 = function (params) {
 			) {
 				throw new Error('Invalid Exchange type pased');
 			}
+
+			if (mode === MODE.Depth && exchangeType !== EXCHANGES.nse_cm) {
+				throw new Error("Depth 20 only supports NSE_CM");
```
Comment (anurag-amx) [inline lib/websocket2.0.js:126]: Correct the sentence Depth 20 only supports NSE_CM.

**Review by anurag-amx — CHANGES_REQUESTED** [review]

File: example/test.js (hunk 1 above)
Comment (Amol-Bhagwat-187) [reply example/test.js]: This code has been uncommented.

File: lib/websocket2.0.js (hunk 2 above)
Comment (Amol-Bhagwat-187) [reply lib/websocket2.0.js]: Sentence has been changed.

**Review by anurag-amx — APPROVED** [review]

**Pull request merged**